en/index.md

```diff
@@ -35,24 +35,23 @@ For **localhost** deployments the following setup is recommended:
 - 2 cores
 - 16G RAM
 - 32G Swap
-- 250G SSD with a separate block device for ZFS. Our recommendation for that
-  device is 100G.
+- 250G SSD
 
 ## Installing conjure-up
 
 `conjure-up` is available on Ubuntu Xenial 16.04 LTS and macOS
 
 ### Ubuntu
 ```bash
-sudo snap install conjure-up --classic
+$ sudo snap install conjure-up --classic
```

Comment:
The docs team has standardised on omitting the prompt and relying on design if that's a requirement.